assets/vendor/css/prism.css

```diff
@@ -1,5 +1,5 @@
-/* PrismJS 1.13.0
-http://prismjs.com/download.html#themes=prism-tomorrow&languages=markup+css+clike+javascript+abap+actionscript+ada+apacheconf+apl+applescript+c+arff+asciidoc+asm6502+aspnet+autohotkey+autoit+bash+basic+batch+bison+brainfuck+bro+cpp+csharp+arduino+coffeescript+clojure+ruby+csp+css-extras+d+dart+diff+django+docker+eiffel+elixir+elm+markup-templating+erlang+fsharp+flow+fortran+gherkin+git+glsl+go+graphql+groovy+haml+handlebars+haskell+haxe+http+hpkp+hsts+ichigojam+icon+inform7+ini+io+j+java+jolie+json+julia+keyman+kotlin+latex+less+liquid+lisp+livescript+lolcode+lua+makefile+markdown+erb+matlab+mel+mizar+monkey+n4js+nasm+nginx+nim+nix+nsis+objectivec+ocaml+opencl+oz+parigp+parser+pascal+perl+php+php-extras+sql+powershell+processing+prolog+properties+protobuf+pug+puppet+pure+python+q+qore+r+jsx+typescript+renpy+reason+rest+rip+roboconf+crystal+rust+sas+sass+scss+scala+scheme+smalltalk+smarty+plsql+stylus+swift+tcl+textile+twig+tsx+vbnet+verilog+vhdl+vim+wiki+xeora+xojo+yaml&plugins=autolinker+toolbar+show-language */
+/* PrismJS 1.16.0
+https://prismjs.com/download.html#themes=prism-tomorrow&languages=markup+css+clike+javascript+ada+apacheconf+applescript+c+csharp+bash+basic+cpp+arduino+coffeescript+clojure+ruby+css-extras+diff+markup-templating+docker+elixir+elm+erb+erlang+flow+gherkin+git+go+graphql+groovy+less+handlebars+haskell+http+java+scala+php+jolie+json+markdown+lua+makefile+crystal+django+typescript+nginx+objectivec+perl+sql+properties+scss+puppet+python+r+js-extras+jsx+reason+rust+sass+smalltalk+plsql+swift+yaml+haml+tsx+regex+vim&plugins=toolbar+show-language */
```

Comment:
Nothing jumps out to me as missing. If so, we can always add in the future. Nice optimization!